src/renderer/atlas/AtlasEngine.h

```diff
@@ -394,14 +394,17 @@ namespace Microsoft::Console::Render
         struct FontMetrics
         {
             wil::com_ptr<IDWriteFontCollection> fontCollection;
-            wil::unique_process_heap_string fontName;
+            std::wstring fontName;

```

Comment:
whhhyy?

Reply:
`wil::unique_process_heap_string` is not copyable and I didn't want to be overly complex. I mean it's 4x larger without benefits (`wstring` only stores 7 characters inline after all and even something trivial like "Consolas" is longer than that), but that doesn't really matter much here I think.